core/domain/collection_services.py

```
@@ -1605,17 +1528,9 @@ def load_demo(collection_id: str) -> None:
-            to be indexed.
+        collection_ids: list(str). List of collection ids.
     """
-    collection_summaries = get_collection_summaries_matching_ids(collection_ids)
-    search_services.index_collection_summaries(
-        [
-            collection_summary
-            for collection_summary in collection_summaries
-            if collection_summary is not None
-        ]
-    )
+    pass
```

Ruff (0.16.1)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)